core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/ChangeEventListener.java

```diff
@@ -17,14 +17,14 @@
 package com.mongodb.stitch.core.services.mongodb.remote.sync;
 
 import com.mongodb.stitch.core.services.mongodb.remote.ChangeEvent;
-
 import org.bson.BsonValue;
 
 /**
  * ChangeEventListener receives change event notifications.
  * @param <DocumentT> the type of class represented by the document in the change event.
  */
-public interface ChangeEventListener<DocumentT> {
+public interface ChangeEventListener<DocumentT>
```

Comment:
[nit] this is clean, but not clear to users since they shouldn't need to have to look into an internal type to see what the interface should look like. Do what you did with `CompactChangeEventListener` and just add the method signature.